root/azuredeploy.json

```diff
@@ -73,8 +73,7 @@
     }
   },
   "variables": {
-    "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'adsa')]",
-    "templateStorageAccountName": "[concat(uniquestring(resourceGroup().id), 'data')]",
+    "storageAccountName": "[concat('adsa', uniquestring(resourceGroup().id))]",
```

Comment:
Good idea to prepend the name. Maybe to keep the naming more consistent with the VM we're making it for, should we prepend 'vmdc' instead of 'adsa'?

Reply:
I used adsa because it was used as subfix. and I thought it means for ad machine. Ya,  I agree vmdc is more suitable for this case.